### DISCUSSION TITLE
How to reduce the file size of the extracted html?

### DISCUSSION THREAD

**Discussion by DJay921** [pymupdf/PyMuPDF#1554] [Q&A]
The quality of the extracted html output for `PyMuPDF `is far better than what I was getting using some of the other libraries like `PDBox `wrapper for python. However, one concern I have is regarding the output file size which is quite larger (1.5 MB) as compared to the other option (400 KB). I am using the flag to skip images using `not fitz.TEXT_PRESERVE_IMAGES `. Apart from this, how can I further reduce the size of the output html file? I'm looking for minified versions of the html code. Thanks. I want to preserve the whitespaces if possibly since the PDF contains a few tables as well.

**Comment by JorjMcKie** [comment]
This is a thin wrapper of an original MuPDF function. So there is no way for me to influence the output, sorry.
Maybe there are postprocessors on the market, that offer syntax optimizations (tidy?), but I really don't know much about this area.

**Reply by DJay921** [reply]
Sure, thanks @JorjMcKie
